guide/payments/send.md

```diff
@@ -48,7 +48,7 @@ Let us look at the main transaction options senders need to configure when movin
 
 You do not need to follow the order below. Feel free to tailor the configuration's order for the payment to what best serves your users. For example, you may make users set the amount before they enter the address.
 
-## Get the recipient address
+## Get the receivers address
```

Comment:
I don't think there's a material benefit in using receiver over recipient.
The whole chapter (and elsewhere in the guide often uses recipient.